lite-node/src/config.rs

```diff
@@ -0,0 +1,56 @@
+//! LiteNode Config
+//!
+//! See instructions in `commands.rs` to specify the path to your
+//! application's configuration file and/or command-line options
+//! for specifying it.
+
+use serde::{Deserialize, Serialize};
+use std::time::Duration;
+
+/// LiteNode Configuration
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(deny_unknown_fields)]
+pub struct LiteNodeConfig {
+    /// RPC address to request headers and validators from.
+    pub rpc_address: String,
+    /// The duration until we consider a trusted state as expired.
+    pub trusting_period: Duration,
+    /// Subjective initialization.
+    pub subjective_init: SubjectiveInit,
```

Comment:
We need to figure out a clever way to represent `TrustThreshold` as sth that can be written to a file. It's currently just defined via it's behaviour.